test/e2e/kubectl.go

```diff
@@ -162,7 +163,12 @@ func runKubectlRetryOrDie(args ...string) string {
 
 var _ = framework.KubeDescribe("Kubectl client", func() {
 	defer GinkgoRecover()
-	f := framework.NewDefaultFramework("kubectl")
+	options := framework.FrameworkOptions{
+		ClientQPS:    20,
+		ClientBurst:  50,
+		GroupVersion: &unversioned.GroupVersion{Group: batch.GroupName, Version: "v2alpha1"},
+	}
+	f := framework.NewFramework("kubectl", options, nil)
```

Comment:
Since we're using the default `ClientQPS` and `ClientBurst` settings in `NewDefaultFramework` here. Suggest adding something like `NewDefaultGroupVersionFramework(baseName string, groupVersion *unversioned.GroupVersion)` which invokes `NewDefaultFramework` and then sets `f.options.GroupVersion`, and we can call this function here and in test/e2e/scheduledjob.go. WDYT?